rstudio/rstudio
# Adding favicon to rstudio user guide

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

The favicon.svg file was never added to the images directory, therefore, it was not displaying.

Related to issue ##12482

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

- Added favicon.svg
- Updated entry in quarto.yml
- Built docs locally `quarto preview`
- Verified that it was displaying:


<img width="1401" alt="2023-03-15_20-47-07" src="https://user-images.githubusercontent.com/31460023/225480619-544e2492-271b-4db5-a937-39935089880d.png">

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

- cd into rstudio user guide docs directory
- run quarto preview
- Verify that favicon displays on tab

### Documentation
> Specify which documentation has been added or modified and why (User Guide? Admin Guide?). If no documentation was added for a new feature, indicate why. 
User Guide

### Checklist
